README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
 res, mAP = tube_evaluator.evaluate(thr=0.5)
 ```
 
+### Running CLI
+
+If there is a need for a command line interface (CLI), you can run the `cli.py` file.
+
+#### CLI Arguments
+The following table shows all the possible arguments you can provide to the CLI:
+| Argument        | Description                                                                                                                                           | Example                                                      | Default | Notes                                                                                                                  |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------|------------------------------------------------------------------------------------------------------------------------|
+| --help, -h      | Shows a help message for tool                                                                                                                         | python cli.py -h                                             |         |                                                                                                                        |
+| --anno_gt       | Takes in a directory or file to the ground-truth annotations                                                                                          | python cli.py --anno_gt toyexample/gts_vocpascal_format      |         | Based on the `gtformat`, make sure this parameter points to the right kind of ground truth (directory or file)         |
+| --anno_gt       | Takes in a directory or file to the detection annotations                                                                                             | python cli.py --anno_gt toyexample/dets_classname_rel_xyx2y2 |         | Based on the `detformat`, make sure this parameter points to the right kind of ground truth (directory or file)        |
+| --img           | Takes in a directory where the corresponding images are located                                                                                       | python cli.py --img toyexample/images                        |         | Used in some ground truth annotation formats. At the current moment, there is visualization functionality for the CLI  |
+| --gtformat      | Takes in keyword that represents the format that the groundtruth will be in (example: COCO GT JSON format)                                            | python cli.py --gtformat coco                                |         | Available formats: `coco`, `voc`, `imagenet`, `labelme`, `openimg`, `yolo`, `absolute`, `cvat`, `tube`                 |
```

Comment:
As other arguments separate the words by "_", shouldn't it be `--gt_format`... or better: '--format_gt` ?

Reply:
I agree on the separation of the keywords, and I'm fine with changing both ground truth and detection to `--format_...`

Reply:
https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/b7cb0e9e74c1f3f27e90eb83a61db86a3798605c